js/components/graph/Graph.js

```diff
@@ -338,6 +341,77 @@ export default class Graph extends React.Component {
     }
   };
 
+  /**
+   * Returns the position of the mouse adjusted to the viewbox's new origin. This is so that the change
+   * in viewboxX and viewboxY are relative to the shifted viewbox origin after panning.
+   * @param {double} clientX
+   * @param {double} clientY
+   * @return {Object}
+   */
+  adjustCoordsToViewbox = (clientX, clientY) => {
+    return {
+      adjustedX: clientX + this.state.viewBoxPos.x,
+      adjustedY: clientY + this.state.viewBoxPos.y
+    };
+  }
+
+  /**
+   * Initializes the panning process by recording the position of the mouse pointer and adding event listeners.
+   * @param {*} event
+   */
+  onMouseDownInGraph = event => {
+    document.body.style.cursor = "grab";
```

Comment:
You can use css for this. Update the `render` method to add a class to one of the container elements when panning, and then use the `cursor` css rule.